.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 parameters:
   machine_image:
     type: string
-    default: ubuntu-2004:202201-02
+    default: 'ubuntu-2204:2023.07.2'
```

Comment:
Why not use `ubuntu-latest`?

Reply:
New commit pushed to use `ubuntu-latest`